Marlin/Marlin_main.cpp

```diff
@@ -1906,13 +1907,16 @@ static void clean_up_after_endstop_or_probe_move() {
 
     if (!endstops.z_probe_enabled) return;
 
+    // Make room for the probe
+    do_probe_raise(Z_RAISE_BETWEEN_PROBINGS);
```

Comment:
I notice that `gcode_G29` for ABL does expect the last raise to always be `Z_RAISE_AFTER_PROBING`, and it only calls `stow_z_probe` or `probe_pt` (which calls `stow_z_probe`). Either we need to ensure that the final raise will always be `Z_RAISE_AFTER_PROBING` or we need to change this calculation:
```cpp
current_position[Z_AXIS] = -zprobe_zoffset + (z_tmp - real_z)
  #if HAS_Z_SERVO_ENDSTOP || ENABLED(Z_PROBE_ALLEN_KEY) || ENABLED(Z_PROBE_SLED)
     + Z_RAISE_AFTER_PROBING
  #endif
;
```